\chapter{Range expansions over heterogeneous landscapes: the interaction of expansion load and migration load}%Mutation load, migration load, expansion load}
\label{chap:expansionload}

\section{Abstract}

\section{temporary section - our questions}

\begin{itemize}

\item How much expansion load should we expect with reasonable biological parameters, even with hard selection
Do we see as much as Peischl et al? If not, why? mutations, disperal, density?
Can we make the case that exp. load is not a problem except at the extreme edge?

are there more deleterious homozygotes in recently colonized pops?

\item How do environmental gradients and expansion load interact to determine rate of range expansion?
is rate of expansion affected by env gradient (yes)
does load ever create a range limit (no)
do adaptation to env gradients and expansion load interact to determine rate of range expansion? (seems yes)

(Note:  for these questions we want to separate the effects of range-wide mutation load from increased load 
caused by drift at the range margin.)

Do for just the cases we had + a case we find where there’s more expansion load alone
	Anova to compare the interactions
	
\item Does one type of load affect the amount of other types of load?

is migration load changed by expansion load / mutation load?
is expansion load changed by migration load?
	response variable is either type of load, based on the amount of the other type of load present
	
we can hypothesize that a steeper gradient slows expansion, so less load
we can alternatively hypothesize that a steeper gradient means a smaller Ne at the edge causing more drift and more load
	second is not true b/c inds die, so first seems to be the case
	
calculate the avg deleterious fitness at our comparison points

\item if there are interactions in the types of load, why? what is the mechanism?
if one type slows the expansion, the other benefits
with fecundity-adjusted runs - what happens?
more mut load overall is known to make everything worse and slower spread and lower Ne
because density is reduced by both, both types of load may become worse
when do individuals at the range edge hae significantly lower fitness than those at the range core? (with gradient and/or mutation load)

What combination of gradient steepness and expansion load yields the highest/lowest fitness at the range edge vs. core? 
	Total fitness edge vs core (not split delet and quanti)

How long does reduced edge fitness persist post colonization?
	? not sure if we want to look at this? Rate of recovery per 18 scenarios?

Does an environmental gradient increase edge fitness, by limiting the pace of range expansion and therefore the degree of expansion load?
If we could get at it easily, a really nice extension of this question would be: 

\item spatial patterns in load?

\item deleterious mutations
	is load due to different effect sizes?
	does surfing happen due to 1 or many loci?
	and what are the effect sizes of surfing loci?

\end{itemize}


\section{Introduction}

Species range expansions are a complex process that is widely studied in evolutionary biology. Because many factors can contribute to range edges and the ability or failure of a population to expand, it is difficult to predict species abilities to expand and adapt into novel environments. Many factors both ecological and evolutionary are known to contribute to range expansions, such as species competition (cite Case and Taper and other empirical papers?), dispersal limitations (cite?), \emph{\small{other examples??}}. An rapidly growing field of research in this area has focused however on the contributions of expansion load to range expansions and the resulting state of recently expanded populations (cite lots - Peischl...). We here investigate two major factors contributing to species range expansions: migration load and expansion load and their potential to interact.

Range expansions create an intriguing evolutionary situation for evolutionary biology. The demography of a range expansion is one in which serial founder events of new colonizing populations at the range front result in persistent reduced effective population sizes at the front. These reductions in \emph{N}$_e$ reduce the efficacy of selection and increase the strength of random genetic drift. As this continues within the expanding wave front, mutations that arise are more likely to drift, resulting in a process termed gene surfing wherein mutations of much larger effect sizes escape selection to which they would otherwise be subject and have the potential to fix or be lost in the edge population \citep{Klopfstein:2006}. This exact process has been posited as a reason for the accumulation of deleterious alleles in populations having undergone recent geographic expansions, in particular for studies of human diseases in populations of Quebec and Scandinavia (cite papers).

Peischl and Excoffier (cite) have described expansion load as the reduction in fitness of range edge populations due to gene surfing. \emph{\small{elaborate more...}}

Previous studies have also examined the impact of environmental gradients on the ability of species ranges to expand. A foundational study by \citet{Kirkpatrick:1997} showed that steeper environmental gradients reduce the ability of expansion due to failure to locally adapt as a result of migration load. For a population existing within a species range and on a linear environmental gradient, symmetric migration from either adjacent population will contribute alleles locally maladaptive, yet this symmetry on weaker gradients still allows local adaptation to succeed. Whereas at a range edge, immigration is asymmetric causing edge populations to diverge from their local optimum in the direction of immigrant individuals. When migration rates are high enough, the edge population fails to adapt and when the environmental gradient is steep enough goes extinct shrinking the entire species range to eventual extinction as asymmetric migration at the edge continues. Apart from asymmetric migration, extreme steep environmental gradients alone can result in too great a migration load that again the species goes extinct. Further studies (cite more) have improved upon this result adding additional biological realism  to models of range expansion... yet strong evidence of evolutionary factors causing a distinct range edge has  not been found.

The combination of these two effects, migration load from underlying environmental gradients and expansion load from genetic drift of deleterious mutations at range edges, has not previously been studied. We investigate here the role of each of these factors in isolation as well as in combination. Using individual-based simulations on a two dimensional, spatially explicit, and approximately continuous landscape, we compare the reductions in fitness (load) that populations experience during range expansions over a range of environmental gradients and with a range of deleterious mutational parameters. Under our range of biologically realistic parameters, we comment on the implications our results have in terms of understanding the presence and persistence of genetic disease in humans. 
\emph{\small{remove that last sentence, or move?}}



\section{Methods}

\subsection{Model}

We model diploid, monoecious individuals undergoing a range expansion over a two dimensional landscape grid using the program \textsc{nemo} \citep{Guillaume:2006}. Simulations were initiated in the left-most 40$\times$40 cells of the landscape and, after burn-in to mutation-selection equilibrium, expanded onto the remaining 1,960$\times$40 cells of the landscape. Each cell on this grid had a maximum carrying capacity of 6 individuals. The simulation software \textsc{nemo} was modified to incorporate a breeding window (code available at \url{https://github.com/kjgilbert/NemoDispersalKernel}), increasing local effective population size above this carrying capacity \emph{\small{anyone want to do a calculation here for that Ne?}}. 

Life cycle events occur in the order of breeding, dispersal, selection, regulation. The breeding window enabled every individual to search for a mate in their own and surrounding cells based on a Gaussian probability kernel. This kernel is parameterized by $\sigma_{breed} = 0.5$ cell widths, where an individual's focal cell is the most likely source of a mate and the 12 surrounding cells can be searched for a mate with decreasing probability. When an individual fails to find a mate, it selfs, an event which most often happens at the expanding edge and mimics the ability of many plant species to self under conditions of pollen limitation (citations). Fecundity was drawn from a Poisson distribution with mean 7.

Dispersal also occurs according to a specified kernel. We tested two types of dispersal kernels: a Gaussian kernel and a leptokurtic kernel, to test the effects of rare, long-distance dispersal events. $\sigma_{disperse}$ for the Gaussian dispersal kernel was set to equal 2 cell widths and the maximum dispersal distance was capped at $8 \times \sigma_{disperse}$. The leptokurtic dispersal kernel was created by summing two different Gaussian kernels with different weightings and calculated to have the same average dispersal distance as our Gaussian kernel: kernel one had a weighting of 0.89 and a $\sigma_{disperse}$ of 1.5 while kernel two had a weighting of 0.11 and a $\sigma_{disperse}$ of 5.5 to create a total kurtosis of 10 for this distribution. The kurtotic kernel was cut off to have the same maximum dispersal distance possible, which in both cases meant an individual could at most disperse 16 cells in one direction from its natal cell. Borders of the 40$\times$2,000 landscape were absorbing, so any individual migrating beyond the edge was removed. R code for calculating and discretizing the dispersal and breeding kernels is available in a wrapper package written for \textsc{nemo}, \textsc{anemone}, available at: \url{https://github.com/kjgilbert/aNEMOne}.

Individuals are characterized by both a quantitative trait underlain by 100 continuous loci and 1,000 deleterious loci. These loci were placed randomly on a genetic map of 10 chromosomes of 100cM each for each simulation replicate. The quantitative trait was under stabilizing selection, with $W(z) = exp[-\frac{1}{2}(z - \theta)^T\omega^{-1}(z-\theta)]$ where $\theta$ is the environmental optimum, and $\omega$ is the ``variance-covariance matrix of selection describing the individual fitness surface``, according to \textsc{nemo}. This selection variance, $V_S$, was set to 7.5. The mutation rate for quantitative loci was set to 0.0001 and the variance in effect size per mutation to 0.02. Environmental variance was set to 1. There is evidence that $\frac{V_S}{V_P} \approx 5$ on average (see Kingsolver et al. 2001; Johnson and Barton 2005). Given that $V_P = \frac{V_E}{1 - h^2}$, $V_S = \frac{5V_E}{1 - h^2}$, so for a heritability of $\frac{1}{3}$ we have $V_E = 1$ and $V_S = 7.5$.

\emph{\small{is it relevant to say the quanti traits were initiated at the mean landscape optimum of the burnin area, since they're not there once the burnin finishes?}}

\emph{\small{is the seln eqn I gave above meaningfully different from the one Nathaniel wrote in the google doc? 
``The fitness associated with a value of z is given by
$\omega_z = exp[-(z-\theta z) \frac{2}{2V_S}]$
where VS is the inverse strength of stabilizing selection``}}

Genomic mutation rates for deleterious loci was varied from 0.0 to 0.1 to 1.0, which for our cases of 1,000 loci were forward mutation rates of 0.0, $5\times10^{-5}$, and $5\times10^{-4}$, with back-mutation rates of 0.0, $5\times10^{-7}$, and $5\times10^{-6}$. A reasonable starting point for the deleterious mutation rate per diploid genome is U = 1.0. Though this may be an underestimate for humans (Keightley 2012), it is about right for Drosophila, (Haag-Liautard et al. 2007), C. elegans (Denver et al. 2004) and possibly non-human endothermic vertebrates (Baer et al. 2007), and probably an overestimate for many other organisms (Baer et al. 2007, Halligan and Keightley 2009). Selection on deleterious loci was direct, and we used mutation model 2 in \textsc{nemo} which places new mutations regardless of existing mutations. The distribution of deleterious mutation effects followed a gamma distribution with mean 0.01 and shape 0.3, and a dominance coefficient of 0.3. Though the true distribution of mutational fitness effects is unclear and is likely to be more complex than a gamma distribution, there is evidence that most mutations have small effects on fitness, and rare mutations have large effects on fitness (Eyre-Walker and Keightley 2007). 




\emph{\small{Flag *** our recessive lethals have 0 heterozygote effect which is unrealistic***
We will assume (as Nemo does) that h is directly proportional to s, with $h = \frac{exp(-ks)}{2}$, where 
k is a scaling factor that relates average dominance to average selection as $k = -\frac{log(2E[h])}{E[s]}$. 
If the average mutation is partially recessive, i.e., E[h] = 0.3, this would suggest k = 10. 
With the gamma parameters above, this leads to E[hs] < 1\%.}}


We compared three underlying environmental gradients, all of which change linearly only along the axis of expansion from one end of the landscape to the other. Gradient values, \emph{b}, of 0.0 (no gradient), 0.0375, and 0.375 are cases where an average dispersal distance for an individual would result in a reduction 
in fitness of 0\%, 0.5\%, or 5\%. All parameter values and their combinations are listed in table \ref{tab:paramresults}. Twenty independent replicates were run for each of our scenarios. Two additional scenarios with no deleterious mutations (U 0.0) but with lowered fecundity to match the realized fecundity in cases with deleterious mutations (U 0.1 and U 1.0) were run under the Gaussian dispersal model. These simulations provide control cases to solely compare the effect of expansion load by mimicking the reduction in fitness due to mutation load alone. Fecundity was reduced by calculating the equilibrium fitness due only to deleterious mutations in the core landscape populations and reducing mean fitness by this factor. The fecundity-adjusted run to match U 0.1 had a mean fecundity of 6.4749923 and the fecundity-adjusted run to match U 1.0 had a mean fecundity of 3.7210544.




\subsection{Analyses}

Our combination of environmental gradients and deleterious mutation rates allow for direct comparisons to be made on the effects of migration load and expansion load independently as well as their interactions. We assess the impact of these processes on two characteristics of the simulations: speed of expansion and fitness, in total as well as broken down into its contributing factors of fitness resulting from only deleterious mutations and fitness for the quantitative trait. Because these fitness measures change over time, we chose an equivalent point at which to measure the populations and make this comparison. Time-lapse GIFs of these results can be seen in supplemental figures S1-Sn. The point at which all fitness measures are compared was the closest time point within 100 generations of any individual existing in the right-most 20$\times$40 cells of the landscape. Because individuals can at most disperse 16 cells, this ensured that no edge effects would be encountered upon filling the landscape, and that sufficient expansion had occurred to accumulate expansion load. We averaged across the axis perpendicular to the direction of expansion, as we saw no major variation within these cross-sections of the landscape, similar to previous studies in two dimensions (cite...).

We defined the landscape core as the first 40 cross sections within which the simulation burn-in took place. We defined the expanding edge of the populations as the inner-most occupied cross section where population size (averaged within that cross-section) was at or below 50\% of the average core population's size. Speed of expansion tracked the time it took the population edge to expand over the landscape from the end of the burn-in until reaching the second to last landscape cross section. Fitness measures in the edge included all individuals within this 50\% edge cross section as well as all individuals present beyond this edge to the point of empty landscape. We compared fitness in these edge populations to fitness in the core to standardize our comparisons to the equilibrium fitness in each scenario. We tested the effects of each parameter combination on these measures with ANOVA in R.

We additionally investigated the details of deleterious mutations accumulating and surfing on the expanding population front. Effect sizes of deleterious mutations were classed into quartiles based on our known gamma distribution of effects. The reduction in fitness from each of these quartiles was then examined to assess if mutations of larger or smaller effect contribute most to expansion load. \emph{need more here on any stats we want to do with these?}
 


\section{Results}

Steeper environmental gradients were shown to slow the speed of expansion (F = 1488, df = 2, p-value < 0.001; Figures \ref{fig:speed}). The leptokurtic dispersal kernel resulted in faster expansion over the landscape (F = 246, df = 1, p-value < 0.001), and increasing the deleterious genomic mutation rate slightly, but significantly decreased the speed of expansion (F = 162.1, df = 2, p-value < 0.001). We find a significant interaction of environmental gradient with the presence of deleterious mutations on expansion speed (F = 62.46, df = 4, p-value < 0.01), where increasing \emph{b} and U result in slower expansion than increasing either individually. All two-way and three-way interactions had a significant effect on expansion speed (p-value < 0.001 for all comparisons).

\emph{need to check I've done stats right - do I want to control for random effects of the variable not of interest in each case, or do them all as one big model like: aov(expansion.speed $\sim$ b*U*disp, data=exp.speeds)}

In our cases comparing fecundity-adjusted runs to cases with deleterious mutations, speed was minimally, but significantly, different for the comparison to U 0.1 (F = 26.73, df = 1, p-value < 0.001; compare dotted line in U 0.0 panel of figure \ref{fig:speed} to solid line in U 0.1 panel), and more significantly different for the comparison to U 1.0 (F = 275.9, df = 1, p-value < 0.001; compare dashed line in U 0.0 panel of figure \ref{fig:speed} to solid line in U 1.0 panel). In these cases, expansion was slower in the cases with deleterious mutations than in those with none but a lowered fecundity to match equilibrium mutation load.


\begin{figure}[]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.9\linewidth]{Figures/expload_speed.pdf}}
\caption[Speed of range expansion across scenarios.]{Speed of range expansion across scenarios measured as cross sections the edge crosses per generation.}
\label{fig:speed}
\end{figure}


Comparing the ratio of fitness in edge populations to core populations showed significant reductions in fitness at the range edge due to steepness of the environmental gradient (F = 2982.389, df = 2, p-value < 0.001) and deleterious mutation rate (F = 29.473, df = 2, p-value < 0.001). Migration load and expansion load significantly interact in a non-additive manner to alter the ratio of fitness at the range edge versus in the range core (F = 306.203, df = 4, p-value < 0.001). Unlike for the expansion speed, there were no other significant interactions of environmental gradient with dispersal mode nor with deleterious mutation rate, nor a three-way interactions of all of these effects on the ratio of total fitness at the range edge versus the range core. All raw fitness values are shown in table \ref{tab:paramresults}.

\emph{do we want a figure of total fitness across all our cases? I have it if the answer is yes}

By parsing total fitness into each of its quantitative trait and deleterious loci fitness components, we then further investigated the mechanism of fitness reduction during range expansion. Not unexpectedly, we found that increasing the steepness of the gradient increases migration load in populations. In the cases for U = 0.0, migration load, and any expansion load doe to maladaptive quantitative locus mutations are the only contributing factors to reduced fitness.

\begin{figure}[]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.9\linewidth]{Figures/expload_load.pdf}}
\caption[Ratio of edge population fitness to core population fitness across scenarios.]{Ratio of edge population fitness to core population fitness across scenarios.}
\label{fig:load}
\end{figure}




\begin{table}[]
\centering \scriptsize
\caption[Fitness and expansion speed results per scenario parameter combinations]{Fitness and expansion speed results per scenario parameter combinations. Expansion speeds are shown +/- 1 standard error and fitness measures are shown at the core and the edge.}
\label{tab:paramresults}
\begin{tabular}{p{0.08\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}|p{0.135\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}}
Environ- mental  		& Genomic Deleterious & Dispersal Kernel & Mean Expansion & Total $\bar{\omega}$	   & Quantitative Trait $\bar{\omega}$ & Deleterious Loci $\bar{\omega}$ \\
Gradient (\emph{b})	& Mutation Rate (U) 	  &	  & Speed 			&	\tiny{Core/Edge}	&	\tiny{Core/Edge}	&	\tiny{Core/Edge}	\\ \hline \hline
0.0                       	&	0.0	& Gaussian         &	3.741 +/- 0.005	&	0.932/0.932	&	0.932/0.932	&	1/1	\\
0.0375                    	&	0.0	& Gaussian         &	2.314 +/- 0.011	&	0.929/0.399	&	0.929/0.399	&	1/1	\\
0.375                     	&	0.0	& Gaussian         &	0.756 +/- 0.003	&	0.870/0.199	&	0.870/0.199	&	1/1	\\ \hline
0.0                       	&	0.1	& Gaussian         &	3.559 +/- 0.008	&	0.863/0.748	&	0.933/0.933	&	0.925/0.802	\\
0.0375                    	&	0.1	& Gaussian         &	2.283 +/- 0.011	&	0.859/0.390	&	0.929/0.461	&	0.924/0.845	\\
0.375                     	&	0.1	& Gaussian         &	0.744 +/- 0.002	&	0.804/0.193	&	0.870/0.217	&	0.924/0.891	\\ \hline
0.0                       	&	1.0	& Gaussian         &	2.285 +/- 0.014	&	0.496/0.306	&	0.933/0.945	&	0.532/0.323	\\
0.0375                    	&	1.0	& Gaussian         &	1.991 +/- 0.011	&	0.494/0.268	&	0.929/0.694	&	0.532/0.387	\\
0.375                     	&	1.0	& Gaussian         &	0.595 +/- 0.002	&	0.463/0.160	&	0.870/0.366	&	0.532/0.437	\\ \hline
0.0                       	&	0.0	& Leptokurtic      &	5.689 +/- 0.010	&	0.933/0.929	&	0.933/0.929	&	1/1	\\
0.0375                    	&	0.0	& Leptokurtic      &	2.695 +/- 0.014	&	0.929/0.369	&	0.929/0.369	&	1/1	\\
0.375                     	&	0.0	& Leptokurtic      &	0.736 +/- 0.002	&	0.871/0.183	&	0.871/0.183	&	1/1	\\ \hline
0.0                       	&	0.1	& Leptokurtic      &	5.355 +/- 0.030	&	0.862/0.770	&	0.933/0.936	&	0.924/0.823	\\
0.0375                    	&	0.1	& Leptokurtic      &	2.637 +/- 0.019	&	0.859/0.360	&	0.929/0.420	&	0.924/0.858	\\
0.375                     	&	0.1	& Leptokurtic      &	0.721 +/- 0.002	&	0.803/0.181	&	0.870/0.206	&	0.923/0.881	\\ \hline
0.0                       	&	1.0	& Leptokurtic      &	3.377 +/- 0.045	&	0.498/0.283	&	0.933/0.938	&	0.534/0.302	\\
0.0375                    	&	1.0	& Leptokurtic      &	2.285 +/- 0.014	&	0.496/0.249	&	0.929/0.644	&	0.534/0.388	\\
0.375                     	&	1.0	& Leptokurtic      &	0.577 +/- 0.004	&	0.463/0.162	&	0.870/0.367	&	0.532/0.442	\\ \hline
\end{tabular}
\end{table}

\section{Discussion}


human papers get different answers - because dominance assumptions matter

our results show that we can detect X amount of load due to loci of greater than or equal to effect size of Y

\subsection{Future Directions}
Can we predict when edge vs. central individuals are more fit at and beyond the current range limit?  
Would require a different aspect of the simulation so maybe it‚Äôs not worth it for this paper, but would tie our results directly to prominent ecological/conservation issues with range expansion, e.g. are edge populations worth conserving? Which genotypes should one use for assisted migration? 
	Could be a future direction

evolution of dispersal


%%% Local Variables:
%%% TeX-master: "thesis"
%%% TeX-PDF-mode: t
%%% End:
