\chapter{Range expansions over heterogeneous landscapes: the interaction of expansion load and migration load}%Mutation load, migration load, expansion load}
\label{chap:expansionload}

\section{Abstract}


\section{Introduction}

%broad intro:
Species range expansions are a complex process which are widely studied in evolutionary biology, ecology, and conservation \citep{Chen:2011, Colautti:2013, Hastings:2005, Phillips:2006, Excoffier:2009, Hallatschek:2010}. There are multitudes of factors that can contribute to range edges and the ability or failure of a species to grow and expand, making it difficult to predict species' abilities to expand and adapt into novel environments. Many factors both ecological and evolutionary are known to contribute to or prevent range expansions, such as dispersal limitations\citep{Hargreaves:2014b, Marsico:2009, Hastings:2005}, species competition \citep{Case:2000, Price:2009,Svenning:2014, Louthan:2015}, or the inability to adapt to new conditions \citep{Polechova:2015, Holt:2011, Angert:2008}. A recently growing field of research has focused on the effects of genetic load accumulated during the expansion process \citep{Excoffier:2009, Hallatschek:2010, Peischl:2013, Peischl:2015, Peischl:2015b}, and in particular, the ramifications this has in populations of humans that have expanded out of Africa \citep{Henn:2015, Do:2015, Lohmueller:2008}. In this study, we explore the impact that the presence of heterogeneous selection due to an underlying environmental gradient has on the accumulation of deleterious mutations during a species range expansion and investigate their potential to interact in driving the ability for local adaptation of populations at range edges.

% range expansions:
Range expansions exhibit an intriguing suite of population genetic processes that can change the course of evolution from our expectations of stationary populations. As populations at the expanding front spread, they undergo serial founder events where each new colonization into further territory bottlenecks the founding population and reduces the diversity of what becomes the new edge population and the new source of further colonizations. %The additional factor of each colonization event being driven by individuals already existing at the current range edge compounds this effect. 
These processes create a persistently reduced effective population size at the range edge, %and these reductions in \emph{N}$_e$ 
reducing the efficacy of selection and increasing the strength of random genetic drift. This leads to the process of gene surfing \citep{Klopfstein:2006}, whereby any mutation that arises at the range edge is more likely drift to fixation than it would be otherwise if the efficacy of selection were not reduced. This entirely neutral process can thus lead to the potential loss of beneficial mutations or the increase of deleterious mutations. Beneficial mutations may also be subject to surfing, but the main cause for an increase in frequency of deleterious alleles in recently expanded populations is due to surfing \citep{Excoffier:2009}. The reduction in fitness due to the accumulation of these deleterious alleles is termed the expansion load \citep{Peischl:2013,Peischl:2015}. Furthermore, these deleterious alleles may be of much larger effect sizes than could exist in a population at equilibrium because they have escaped selection to which they would otherwise ordinarily be subject \citep{Peischl:2015}. 

% deleterious mutations:
%Peischl and Excoffier (cite) have described expansion load as the reduction in fitness of range edge populations due to gene surfing. Because deleterious mutations are probabilistically more likely to occur than beneficial mutations, surfing often results in fitness reductions rather than improving from beneficial mutations.
Expansion load has been posited as an explanation for the accumulation of deleterious alleles in populations having undergone recent geographic expansions \citep{Karlsson:2014}, in particular for studies of human diseases in populations of Quebec \citep{Scriver:2001, Yotova:2005, Labuda:1997} and Scandinavia \citep{Norio:2003}%check that this ref is appropriate
. A large point of controversy however, is the likelihood of these deleterious mutations persisting on a timescale that would implicate this process as causing deleterious mutations in humans. It has only recently been shown that as an expansion proceeds, deleterious mutations of larger effect are more likely to surf and rise in frequency in populations \citep{Henn:2015}. Expansion load may thus have the potential for serious repercussions in many species that will or have experienced expansions. Understanding the detriment caused by this process may aid in efforts to combat invasive species or in efforts of assisted migration. Under the effects of global climate change, more species than ever may be required to move their ranges in order to survive, and if accumulating fitness losses simply due to the expansion process is widespread, populations may be left at higher risk for extinction from stochastic, catastrophic events. % nathaniel maybe you could work on this paragraph or we can think of someone else who would be good here.

% environmental gradients:
A major goal of this study is to assess how prevalent expansion load may be by using a unique simulation approach and by parameterizing our model with biologically realistic terms. The introduction of an environmental gradient over which expansion occurs adds biological realism that has not previously been investigated in combination with expansion load. Previous studies have examined the impact of environmental gradients alone on the ability of species ranges to expand. A foundational study by \citet{Kirkpatrick:1997} showed that steeper environmental gradients reduce the ability of expansion due to failure to locally adapt as a result of migration load. For a population existing within a species range and on a linear environmental gradient, symmetric migration from either adjacent population will contribute alleles locally maladaptive, yet this symmetry on weaker gradients still allows local adaptation to succeed. Whereas at a range edge, immigration is asymmetric causing edge populations to diverge from their local optimum in the direction of immigrant individuals. When migration rates are high enough, the edge population fails to adapt and, when the environmental gradient is steep enough, goes extinct shrinking the entire species range to eventual global extinction as asymmetric migration at the edge continues. Apart from asymmetric migration, extreme steep environmental gradients alone can result in too large a migration load that again the species range contracts, potentially to extinction. Further studies \citep{Barton:2001, Polechova:2015} have improved upon this result adding additional biological realism including evolution of genetic variance and effects of genetic drift to models of range expansion, showing that indeed evolutionary processes alone can form a stable range edge. % feel like I have something else to say about migration load here, but can't get it out of my head

% say something about migration load and genetic rescue?


% their interaction and what this study is doing:
It is clear that expansion load can result in reduced fitness of recently expanded populations, and that heterogeneous selection on an environmental gradient can result in local maladaptation of populations across a species range. The combination of these two effects, migration load from underlying environmental gradients and expansion load from genetic drift of deleterious mutations at range edges, has not previously been studied, and the outcome is unclear. One expectation may be the compounded detriment of both of these effects at a range edge: accumulation of deleterious load and reduced adaptation to the local environment exacerbating the reduced fitness at range edges. Alternatively, one may predict the amelioration of each effect on the other: reduced survivability of range edge populations prevents the accumulation of expansion load and allows for genetic rescue through migration. We investigate here the role of each of these factors in isolation as well as in combination. Using individual-based simulations on a two dimensional, spatially explicit, and approximately continuous landscape, we compare the reductions in fitness (load) that populations experience during range expansions over a range of environmental gradients and with a range of deleterious mutational parameters. These results have implications for the predicted prevalence of expansion load in species existing across various types of environments and inform our understanding of the complex demographic and genetic processes that occur during range expansions.




\section{Methods}


% broad summary of what we modeled:
We model a species range expansion forward in time using the simulation program \textsc{nemo} \citep{Guillaume:2006}. The population undergoes an initial burn-in period to mutation-selection equilibrium, after which the population is allowed to expand across empty landscape. Individuals are monoecious and diploid, possessing both a quantitative trait experiencing balancing selection from the environment and a set of loci that can accrue unconditionally deleterious mutations. % under direct selection. 
Our goal is to employ the most biologically reasonable parameters possible within the realm of our model. We compare three different environmental gradients over which expansion occurs, and three different genome-wide mutation rates for deleterious alleles. All combinations of parameter values are listed in Table \ref{tab:paramresults}. We ran twenty independent replicates for each of our simulated scenarios.


\subsection{The Model}

% landscapes
The environmental gradient underlying the landscape changes in only one dimension, along the axis of expansion. We compare three values for the steepness of this gradient, $b$, which defines the change in phenotypic optimum over space: a flat landscape with no gradient ($b = 0.0$), a weak gradient ($b = 0.0375$), and a strong gradient ($b = 0.375$). On the weak (shallow) gradient, an average dispersal distance of an individual would result in a loss in fitness of $0.5\%$ if it were perfectly adapted in its natal environment, whereas on the strong (steep) gradient, an average dispersal event would reduce an individual's fitness by $5\%$ given perfect adaptation in its natal environment. Each of these gradients results in a different equilibrium level of migration load in populations.

% landscape and population details - and general model details
We model space explicitly on a $40\times2000$ unit landscape using a modified version of \textsc{nemo} available at \url{https://github.com/kjgilbert/NemoDispersalKernel}. This modification allows many patches of small carrying capacity to interact with their surrounding patches through both the breeding and dispersal life cycles. Life cycle events occur in the order of breeding, dispersal, selection, and then population regulation. Population regulation enforced a carrying capacity of 6 per unit (patch) of the landscape. Thus, one could imagine our landscape being on the scale of meters or kilometers depending on the density of the organism in mind. % mike wants to remove this sentence, but I kind of like it - other thoughts?
We do not aim to model any specific organism in this study. % mike wants to move this sentence to intro, I like it better here
Because breeding is not limited to within one unit, the realized neighborhood size as defined by \citet{Wright:1946} is approximately 300 individuals and therefore much larger than the specified carrying capacity per unit of the landscape. 

%--------- Remi's calculations ---------
%Here is how I calculated it. Note that I used the original formal definition by Wright but in empirical papers, authors seem to playfully use different definitions!
%The "neighbourhood size" is the number of individuals present in the "neighbourhood area", an area where parents can be considered representative of the offspring. The first one to use this term was Wright. In Wright 1945 (page 41). Wright defines the "neighbourhood area" as the circle of $2 \sigma$ of radius, where $\sigma$ is the standard deviation of the migration distance. The "neighbourhood size", is therefore $\pi * n$, where $n$ is the number of individuals in a big square of $2 \sigma$ on a side. 
%In our case $\sigma = 2$ demes for both the leptokurtic and normal distribution and the population per small deme is 6. As a a consequence $n = (2*2)^2 * 6 = 96$ and the "neighbourhood size" is $96 * pi \approx 301.59$ individuals.
%------ End of Remi's calculations ------

Populations were initiated at carrying capacity in the left-most $40\times40$ units of the landscape, which we term the landscape core (Figure \ref{fig:landscape}). After a burn-in period of 15,000 generations, the remaining $40\times1960$ units of the landscape become available, allowing for expansion to occur. We will later refer to a 40-unit width of the landscape, across which the environmental optimum is constant (perpendicular to the axis of expansion), as a cross-section.

\begin{figure}[h]
\centering
\makebox[\textwidth]{
        \includegraphics[width=1\linewidth]{Figures/Landscape_Graphic.pdf}}
\caption[Landscape graphic.]{Landscape graphic representing an example of the landscape simulated with a grid of units where the burn-in period occurs in the landscape core at the left, expansion proceeds to the right, and any vertical column of units is a cross-section. Actual landscapes were $40\times2000$ units with a $40\times40$ core -- this simplification shows a smaller grid of $35\times150$ for easier visualization.}
\label{fig:landscape}
\end{figure}

\subsubsection{Breeding}
% breeding
Our modified version of \textsc{nemo} creates a breeding window within which individuals may search for a mate within their own unit of the landscape or in nearby units. Mating probability follows an approximate bivariate Gaussian which we derive by integrating the probability over each unit within the breeding window. We define the size of this breeding window with the parameter $\sigma_{breed}$, where $f(x,y) \propto \exp{[-(\frac{\Delta x^2}{2\sigma_{breed}^2}+\frac{\Delta y^2}{2\sigma_{breed}^2})]}$ gives the distance traveled to search for a mate in a given direction $x$ and $y$. %describes the probability for a given distance traveled. 
$\sigma_{breed}$ was held constant at $0.5$ $\times$ one landscape unit.
%This search follows a probability distribution we define by discretizing a continuous, Gaussian probability kernel. This kernel was parameterized by $\sigma_{breed} = 0.5$ $\times$ one landscape unit, where $\sigma_{breed}$ is the standard deviation of the kernel. This continuous kernel was discretized by integrating the probability over each landscape unit, restricting non-zero probabilities within $4 \times \sigma_{breed}$, 2-dimensionalizing these probabilities, %how the heck to describe that besides saying it was multiplied across itself?
The maximum searchable distance was restricted to $4 \times \sigma_{breed}$ after which probabilities were renormalized. This resulted in a two-dimensional breeding window where an individual's focal cell is the most likely source of a mate and the 12 surrounding cells could be searched for a mate with decreasing probability. We then weighted these probabilities by the number of potential mates present within each landscape unit of the breeding window. 
%should I point out that this weighting means we didn't have to treat the edge differently? - besides it affecting the weightings that is
When no other individuals are present within the breeding window, an individual fails to find a mate and self-fertilizes. This will most often happen at the expanding range edge where population densities are lowest and mimics the ability of many plant species to self under conditions of pollen limitation \citep{Hargreaves:2014}. Each female's fecundity was drawn from a Poisson distribution with mean 7 to determine the number of mating events. 

\subsubsection{Dispersal}
% dispersal
Dispersal occurred according to a specified kernel, also defined by an approximate bivariate Gaussian, similarly to the breeding kernel: $f(x,y) \propto \exp{[-(\frac{\Delta x^2}{2\sigma_{disperse}^2}+\frac{\Delta y^2}{2\sigma_{disperse}^2})]}$. The dispersal kernel gives the forward migration probabilities for offspring to disperse to a given cell. Unlike the breeding kernel, the maximum dispersal distance was capped at a distance of $8 \times \sigma_{disperse}$. % after which discrete probabilities were renormalized. 
There was no weighting based on the number of individuals present in a cell. We simulated two different types of dispersal kernels: a Gaussian kernel and a leptokurtic kernel, to test the effects of rare, long-distance dispersal events. $\sigma_{disperse}$ for the Gaussian dispersal kernel was set to equal 2 $\times$ one landscape unit. The leptokurtic dispersal kernel was a mixture distribution created from a weighted sum of two different Gaussian kernels \citep{Ibrahim:1996}. This leptokurtic kernel was calculated to have the same average dispersal distance as the Gaussian kernel and a total kurtosis of 10. This value of kurtosis is not unrealistic for long-distance dispersal in many species \citep{Guttal:2011, Lowe:2009, Skalski:2000}. The first distribution used a $\sigma_{disperse}$ of 1.5 with a weighting of 0.89, while the second used a $\sigma_{disperse}$ of 5.5 and a weighting of 0.11. The kurtotic kernel was cut off to have the same maximum dispersal distance, which in both cases meant an individual could at most disperse 16 units in any one direction from its natal cell. Borders of the $40\times2000$ landscape were absorbing, so any individual migrating beyond the edge was removed. R code for calculating and discretizing the dispersal and breeding kernels is available in a wrapper package written for \textsc{nemo}, \textsc{aNEMOne}, available at: \url{https://github.com/kjgilbert/aNEMOne}.


\subsubsection{Genetics}
% paragraph on quanti traits & selection
We modeled a genetic architecture where 100 quantitative trait loci and 1000 loci subject to unconditionally deleterious mutations were randomly dispersed and interspersed across the genome for each simulation. Each genome consisted of ten chromosomes of 100 cM each, over which all loci were randomly placed. We selected realistic mutational parameters for these loci as follows.

The quantitative trait, $z$, was underlain by 100 continuously distributed, additive loci. The evolution of this trait under stabilizing selection depends on the mutational variance, $V_M$, and the intensity of stabilizing selection on the trait, $V_S$. These properties have been empirically estimated for a number of traits, along with the corresponding environmental variances, $V_E$. For our model we set $V_M$ and $V_S$ relative to the same arbitrary $V_E$ value ($V_E = 1$). There is evidence that $V_S \approx 5 V_P$ on average (where $V_P$ is the phenotypic trait variance; \citet{Kingsolver:2001, Johnson:2005}). The relationship between $V_P$ and $V_E$ can be expressed in terms of heritability, $h^2 = 1 - \frac{V_E}{V_P}$, and we can therefore write $V_S = \frac{5V_E}{1 - h^2}$. Using $V_E = 1$ and a typical heritability of $h^2 \approx \frac{1}{3}$ \citep{Mousseau:1987, Houle:1992}, we set $V_S = 7.5$.  

Most reported values of $\frac{V_M}{V_E}$ are in the range $10^{-4}$ to $10^{-3}$ \citep{Houle:1996}. $V_M$ depends on the genome-wide rate of mutations affecting the trait, $U_z$, and the expected squared effect of a mutation on the trait, $E[\alpha^2]$, where $V_m = U_z E[\alpha^2]$. For a trait under stabilizing selection $E[\alpha] = 0$, and so $E[\alpha^2] = V[\alpha]$, the variance of mutational effects on the trait. These components are difficult to estimate, but according to one theoretical approach we expect $V_P - V_E = 4 U_z V_S$ \citep{Turelli:1984, Charlesworth:2010}. Given the parameters above, this implies $U_z \approx 0.02$, similar to some direct estimates \citep{Lynch:1998}. In our simulations we used 100 quantitative trait loci, with a mutation rate per locus of $1\times10^{-4}$, giving a diploid mutation rate of $0.02$. We further assumed $V[\alpha] = 0.02$ (i.e., mutational effects on $z$ are drawn from a normal distribution with mean 0 and variance 0.02), giving a conservative value for $V_M$ of $4\times10^{-4}$.

In this model an individual's quantitative trait value is given by the sum of allelic effects across all loci, with no dominance. Alleles are continuous, such that a mutation's effect is added to the existing allelic value. The fitness (survivorship) of an individual with trait value $z$ is $\omega_z = exp[-\frac{(z-z_{opt})^2}{2V_S}]$, where $z_{opt}$ is the optimal trait value for the given location on the landscape. At the start of the burn-in, all individuals were initiated with $z$ equal to the mean $z_{opt}$ of the core (burn-in area). 

% add this in as a section distinguisher?  \paragraph{Deleterious Mutations}
We also modeled 1000 bi-allelic loci subject to unconditionally deleterious mutations. We considered genome-wide diploid deleterious mutation rates, $U_D$, of 0.1 and 1.0 in order to encompass probable rates for a variety of taxa, including \emph{Drosophila melanogaster}, \citep{Haag:2007}, \emph{Caenorhabditis elegans} \citep{Denver:2004}, \emph{Arabidopsis thaliana} \citep{Shaw:2000}, \emph{Amsinckia sp.} \citep{Schoen:2005}, and possibly non-human endothermic vertebrates \citep{Baer:2007}. However, we note that in humans $U_D$ likely exceeds 1.0 \citep{Keightley:2012}, and $U_D$ is likely less than 0.1 in many other organisms \citep{Baer:2007, Halligan:2009}. We also included a treatment where deleterious mutations were absent ($U_D = 0.0$). Deleterious mutation rates per haploid locus were thus 0.0, $5\times10^{-5}$, and $5\times10^{-4}$, and we allowed for back-mutation at rates of 0.0, $5\times10^{-7}$, and $5\times10^{-6}$, respectively. % Sites already homozygous for deleterious alleles were not included when randomly selecting loci for forward mutation. <- NOT TRUE, we chose nemo model 2
These 1000 loci do not accurately portray the number of possible loci in biological systems, but are instead an approximation required by simulation. By matching our genome-wide mutation rates to realistic values, each of these loci is more representative of a region of the genome within which a deleterious mutation may arise. Thus, for the distribution of effects we use, small effect mutations may be approaching saturation, but because these small effect loci do not contribute substantially to fitness reductions, this lack of realism in terms of the number of loci is not detrimental to our model and its results.

Though the true distribution of mutational fitness effects is unclear and may be complex, there is evidence that most mutations have small effects on fitness, and rare mutations have large effects on fitness \citep{Eyre:2007}. We modeled the homozygous fitness effects ($s$) of deleterious mutations using a leptokurtic gamma distribution with mean 0.01 and shape 0.3, such that most mutations have $s < 1\%$ \citep{Keightley:1994}. The mutational effect of each locus was drawn from this distribution at the start of each independent simulation run, and remained constant throughout the run.

We modeled the dominance coefficient, $h$, of deleterious allele $i$ as $h_i = exp[-k s_i]/2$, where $k$ is a constant scaling factor, described by $k = -\frac{\log{(2 \bar{h})}}{\bar{s}} \approx 51.1$. This means that $h$ approaches 0.5 (additivity) as $s$ approaches 0, and $h$ approaches 0 (complete reccessivity) as $s$ approaches 1. Given these parameters, our specified $\bar{h} = 0.3$ results in a realized $E[h] \approx 0.37$ and $E[hs] \approx 1.4\times10^{-3}$. This relationship is more realistic than a constant $h$, and captures some of the coarse patterns thought to occur in the relationship between $h$ and $s$ \citep{Agrawal:2011}. Finally, we also included a class of deleterious mutations with $s = 1$ and $h = 0$, i.e., recessive lethals, and assumed that 3\% of deleterious mutations fall into this class to match the genome-wide rate typically observed in \emph{D. melanogaster} \citep{Fry:1999}.


\subsubsection{Mimicking Mutation Load} % get rid of the subsection, or change it's name to something better
We compared an additional parameter set to the core set described above in order to disentangle the effects of mutation load and expansion load. In the presence of deleterious alleles, all simulated populations would experience an equilibrium level of mutation load from the influx of deleterious alleles. % and their subsequent removal through selection. 
During range expansion, the effects of deleterious mutations can increase due to genetic drift during the expansion process, resulting in expansion load. To mimic the presence of only mutation load and no expansion load, we therefore alter individual fecundity to reduce realized fitness to a level that matches the equilibrium mutation load measured in each of our scenarios of $U = 0.1$ and $U = 1.0$ within the population core that has not undergone expansion. This was only done for the cases of Gaussian dispersal. We calculated the equilibrium fitness due only to deleterious mutations in the core landscape populations and reduced mean fecundity to recreate this same realized fitness. Mean fecundity in the case matched to $U = 0.1$ was approximately 6.475 %6.4749923
and approximately 3.721 %3.7210544.
for the $U = 1.0$ case.
% how many sig figs do we want on those numbers? these are the exact ones I put into Nemo



%\subsection{Analyses}
%\color{red}\emph{I think remove this whole section now that the results are revamped to explain better} 
%Our combination of environmental gradients and deleterious mutation rates allow for direct comparisons to be made on the effects of migration load and expansion load independently as well as their interactions. We assess the impact of these processes on two characteristics of the simulations: speed of range expansion and fitness partitioned into its contributing factors of fitness resulting from only deleterious mutations and fitness for the quantitative trait. We maintain these partitions because it is clear that in nature there is more than one quantitative trait contributing to an individual's phenotype, therefore we have no meaningful approach to correctly weight the combination of quantitative trait fitness and deleterious allele fitness. Because the scenarios do expand at different rates, to create an equivalent comparison of fitness on the landscape, we chose to measure populations at the latest recorded point in the simulations before any individuals had dispersed into the last $40\times20$ units of the landscape. %this was accurate to within 100 generations
%Because individuals can at most disperse 16 units, this ensured that no edge effects would be encountered upon filling the landscape, and that sufficient expansion had occurred to accumulate expansion load. We averaged population sizes and fitness within cross-sections of the landscape, as we saw no major variation on this axis.

%To make comparisons at the range edge, we define the start of the edge as the first cross-section away from the core at which population size is at $50\%$ of the core's equilibrium population size. This cross-section alone is used to measure the speed of range expansion across time. Speed of expansion tracked the time it took the population edge to expand over the landscape from the end of the burn-in until reaching the second to last landscape cross section. All individuals within this edge cross-section plus all other individuals further away from the core until empty landscape is reached are included into the populations measured for fitness at the range edge. 
% Fred recommended we have a figure explaining these measures and definitions
%We compared fitness in these edge populations to fitness in the core to standardize our comparisons to the equilibrium fitness in each scenario. % remove this sentence?
%We additionally investigated the details of deleterious mutations accumulating and surfing on the expanding population front. Effect sizes of deleterious mutations were classed into quantiles based on our known gamma distribution of effects. The reduction in fitness from each of these quantiles was then examined to assess if mutations of larger or smaller effect contribute most to expansion load.
%\color{black}


\section{Results}

\subsection{Expansion Speed}
% what are these effects on expansion speed? 
% (because all other interpretations come back to lower fitness reducing speed, I put this first) (also because it is the only result we present with kurtotic dispersal, so makes sense to have it then say we put the rest in supplemental)
No parameter combinations that we investigated ever led to the formation of a stable range edge. However, the rate at which populations expanded across the landscape was impacted by the presence of heterogeneous selection from the environmental gradient and deleterious mutations. We therefore measured and compared the rate of range expansion. We use a single landscape cross-section as the range edge in order to track the expanding front and measure the speed of expansion. This edge was defined as the first cross-section away from the core at which population size is at $50\%$ of the core's equilibrium population size. Speed of expansion is a measure of the distance traveled over the landscape from the end of the burn-in until reaching the second to last landscape cross section, divided by the time taken to do so.

Steeper environmental gradients were shown to slow the speed of expansion (Figure \ref{fig:speed}). Leptokurtic dispersal kernels resulted in faster expansions over the landscape, particularly in the absence of an environmental gradient where expansion speed was 66\% to 76\% greater for kurtotic dispersal. Increasing the deleterious genomic mutation rate slightly, but significantly decreased the speed of expansion (ANOVA $F_{2,358} = 3749.5$,  p $< 0.001$). While steepness of the environmental gradient had large effects on slowing the speed of expansion. The difference between no gradient and the steepest gradient slowed expansion in the Gaussian dispersal cases to 20-23\% in the presence of the steep gradient.

\begin{figure}[h]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.8\linewidth]{Figures/expansion_speed.pdf}}
\caption[Average speed of expansion.]{Average speed of expansion across scenarios, measured as the number of landscape cross sections colonized per generation. Circles indicate Gaussian dispersal kernels while triangles indicate kurtotic dispersal kernels. Dashed lines indicates fecundity-adjusted simulations which replicate the loss in realized fitness due to mutation load, but lack expansion load because no deleterious alleles are present. Error bars indicate 95\% confidence intervals.}
\label{fig:speed}
\end{figure}





\subsection{Fitness and Load}

To compare fitness at the range edge across scenarios as well as to fitness in the range core, we defined a larger region of individuals than the cross-section used to measure speed of expansion. This new range edge included all individuals present in the first cross-section away from the core with population size at $50\%$ of the core's equilibrium population size and extending outward to include all individuals until empty landscape is reached. Because expansion proceeded at different rates, to create an equivalent comparison of fitness on the landscape, we chose to measure populations at the latest recorded point in the simulations before any individuals had dispersed into the last $40\times20$ units of the landscape. %this was accurate to within 100 generations
% we take our measurements of fitness at approximately the same location on the landscape, rather than at the same point in time. 
Since individuals can at most disperse 16 units, this prevented edge effects potentially encountered upon filling the landscape from altering our results. We averaged population sizes and fitness within cross-sections of the landscape, as we saw no major variation on this axis.

% what is the pattern of mean fitness? (spatially)
We partitioned total fitness into fitness due to the quantitative trait's level of adaptation to the local optimum and fitness due to the presence of deleterious alleles. In general, the presence of heterogeneous selection or the presence of deleterious mutations both reduced mean fitness in both the core and edge, as expected. Furthermore, fitness at the expanding edge was always reduced relative to the core of the species range (Figure \ref{fig:fitness}). 

Core quantitative fitness was nearly identical across $U$ cases and ranged in total from 0.87 to 0.93 (Table \ref{tab:paramresults}). Fitness reduction at the edge for the quantitative trait only occurred in the presence of an environmental gradient and ranged from a $25-77$\% reduction in fitness at the edge. The most severe fitness loss at a range edge was for $U = 0, b = 0.375$ where edge quantitative fitness was reduced to $0.2$. %(though on no gradient, $b = 0$, there was not always a reduction, i.e. percent lost ranged from 0.0007 to -0.013)
Fitness for deleterious alleles experienced in the range core was a result of the equilibrium mutation load reached in the respective $U$ cases. For $U = 0.1$, core fitness was $0.92$, while for $U = 1$, core fitness was $0.53$. Edge fitness for deleterious alleles ranged from $0.80-0.89$ for $U = 0.1$ and from $0.32-0.44$ for $U = 1$. Table \ref{tab:paramresults} reports all fitness values for the respective scenarios simulated.
%, was also reduced at the range edge by a range of 39\% on no gradient with $U = 1$ to 3.5\% on the steepest gradient with $U = 0.1$.


\begin{figure}[h]
\centering
\makebox[\textwidth]{
        \includegraphics[width=1.2\linewidth]{Figures/core_edge_fitness.pdf}}
\caption[Average core and edge fitness.]{Average core and edge fitness for quantitative trait and deleterious allele fitness. Dashed lines indicates fecundity-adjusted simulations which replicate the loss in realized fitness due to mutation load, but lack expansion load because no deleterious alleles are present. Error bars indicate 95\% confidence intervals.}
\label{fig:fitness}
\end{figure}


% is there load and how much
This reduction in fitness for deleterious alleles is the measure of expansion load in our simulations. We compared the ratio of fitness in edge populations to fitness in the core to standardize our comparisons to the equilibrium fitness in each scenario. At its worst for our parameter sets, expansion load causes a 39\% reduction in fitness, while at its weakest results in a 3.5\% decrease in fitness (Figure \ref{fig:load}). % this part goes below when talking about the interaction: "We find that expansion load is greatest in the absence of an environmental gradient (Figure \ref{fig:load}). The presence of any environmental gradient significantly reduces expansion load. This effect is greatest on the steepest gradient, where in the case of $U = 1$, there is a 45.7\% reduction in expansion load relative to the case of no gradient. In the case of $U = 0.1$, the load is reduced by the steepest gradient by 26.6\%." 
Across the two $U$ cases, on no gradient, $U = 1$ has a 5.1-fold increase in expansion load over the case of $U = 0.1$ while on the steepest gradient this change in $U$ resulted in a 2.95-fold increase in expansion load.

\begin{figure}[h]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.5\linewidth]{Figures/expansion_load.pdf}}
\caption[Average expansion load.]{Average expansion load across all combinations of environmental gradients and genome-wide deleterious mutation rates. Error bars indicate 95\% confidence intervals.}
\label{fig:load}
\end{figure}


Since kurtotic dispersal only showed meaningful differences for expansion speed, and not for fitness, those results are presented in supplemental figure \color{red}Sn\color{black}. Supplemental figure \color{red}Sn \color{black} presents detailed results on recovery rates for populations after the expansion process has occurred. Furthermore, time-lapse GIFs of fitness and population size across the entire landscape can be seen in supplemental figures \color{red}Sn-Sn\color{black}.


\subsection{Interaction of Expansion Load and Heterogeneous Selection}
% is amount of migration load/lack of local adaptation impacted by expansion load
We next investigate how each component of fitness is affected by the presence of load due to the other component, i.e. does expansion load impact the degree of migration load and local maladaptation, and does local maladaptation impact the degree of expansion load accumulated. First, does the presence of expansion load increase or decrease the amount of adaptation exhibited in the quantitative trait. We find that expansion load plays a role in the level of local maladaptation, but interestingly an increase in load due to deleterious alleles reduces the level of maladaptation (Figure \ref{fig:fitness}a). This becomes clear when we consider the fecundity-adjustment simulations that lack the presence of expansion load. Within a given value of environmental gradient, quantitative trait fitness increases when only the effects of mutation load are present (fecundity-adjusted runs) and increases even further in the presence of both mutation load and expansion load ($U = 0.1$ and $U = 1$). 
%This increase in fitness in the presence of mutation and expansion load is true even in the absence of a gradient, though much less significantly so, where quantitative trait fitness at the range edge increases over that in the core (\emph{by 0.00017\% and by 0.013\%, so maybe remove this sentence?}). 
The largest increase in quantitative trait fitness occurs on the weak gradient ($b = 0.0375$) between $U = 0$ and $U = 1$, where the presence of mutation load leads to an 1.3-fold improvement in quantitative trait fitness on average, and the presence of mutation and expansion load increase quantitative trait fitness by 1.7-fold. % include fig reference again? (Figure \ref{fig:fitness}a.

% is amount of expansion load impacted by migration load/lack of local adaptation
Second, we find that the severity of expansion load is mitigated by the presence of an environmental gradient. We find that expansion load is greatest in the absence of an environmental gradient (Figure \ref{fig:load}). The presence of any environmental gradient significantly reduces expansion load. Within a $U$ case, expansion load decreases with increasing gradient steepness. In the case of $U = 1$, there is a 45.7\% reduction in expansion load from the no gradient to the steepest gradient. In the case of $U = 0.1$, expansion load is reduced by the steepest gradient by 26.6\%. 



\subsection{Loci Contributing to Expansion Load}
% what is the mechanism of expansion load (effect sizes)
We additionally investigated the details of deleterious mutations accumulating and surfing on the expanding population front. Effect sizes of deleterious mutations were classed into quantiles based on our known gamma distribution of effects. We examine the distribution of loci contributing to expansion load at the range edge in our respective cases of $U$ and $b$. For $U = 1$, the majority of expansion load is due to alleles of intermediate effect (Figure \ref{fig:loci}b). On the steep gradient, there is less load overall due to each locus, but a more constant value across loci of both intermediate and large effect. On the weak gradient and with no gradient, there are fewer large effect alleles contributing to load relative to the strong gradient, but also many more intermediate-sized alleles contributing to more overall load. For $U = 0.1$ (Figure \ref{fig:loci}a), there is less overall load, so lower averages within most ranges of allelic effect size, however we see an inflation of larger effect alleles  above that seen for the $U= 1$ case in the absence of an environmental gradient.

\begin{figure}[H]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.9\linewidth]{Figures/load_locibinned.pdf}}
\caption[Average expansion load per locus.]{Average expansion load per locus at the range edge, where loci are binned into equal-sized quantiles drawn from our modeled gamma distribution, shown by the histogram of loci density.}
\label{fig:loci}
\end{figure}

There are very few loci present in these larger effect size classes, yet they still contribute largely to expansion load. As can be seen in Figure \ref{fig:allfreqs}, some of these larger effect loci have fixed at the range edge, and the effect of surfing which contributes to the fixation of these larger effect alleles can be seen in animation over time with supplemental figure \color{red}XX\color{black}, where recovery from fixation follows behind the expanding wave front.

\begin{figure}[h]
\centering
\makebox[\textwidth]{
        \includegraphics[width=0.9\linewidth]{Figures/example_allfreqs.png}}
\caption[Deleterious allele frequencies across the landscape.]{Deleterious allele frequencies across the landscape, binned into classes of homozygote effect size, \emph{s}. This hand-picked example is at 250 generations after burn-in for the case of a kurtotic dispersal kernel, $U = 0.1$ and $b = 0$.}
\label{fig:allfreqs}
\end{figure}

% recovery rates in suppmat




\begin{table}[h]
\centering \scriptsize
\caption[Fitness and expansion speed results per scenario parameter combinations]{Fitness and expansion speed results per scenario parameter combinations. Expansion speeds are shown +/- 1 standard error and fitness measures are shown at the core and the edge.}
\label{tab:paramresults}
\begin{tabular}{p{0.08\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}|p{0.135\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}|p{0.1\textwidth}}
Environ- mental  		& Genomic Deleterious & Dispersal Kernel & Mean Expansion & Total $\bar{\omega}$	   & Quantitative Trait $\bar{\omega}$ & Deleterious Loci $\bar{\omega}$ \\
Gradient (\emph{b})	& Mutation Rate (U) 	  &	  & Speed 			&	\tiny{Core/Edge}	&	\tiny{Core/Edge}	&	\tiny{Core/Edge}	\\ \hline \hline
0.0                       	&	0.0	& Gaussian         &	3.741 +/- 0.005	&	0.932/0.932	&	0.932/0.932	&	1/1	\\
0.0375                    	&	0.0	& Gaussian         &	2.314 +/- 0.011	&	0.929/0.399	&	0.929/0.399	&	1/1	\\
0.375                     	&	0.0	& Gaussian         &	0.756 +/- 0.003	&	0.870/0.199	&	0.870/0.199	&	1/1	\\ \hline
0.0                       	&	0.1	& Gaussian         &	3.559 +/- 0.008	&	0.863/0.748	&	0.933/0.933	&	0.925/0.802	\\
0.0375                    	&	0.1	& Gaussian         &	2.283 +/- 0.011	&	0.859/0.390	&	0.929/0.461	&	0.924/0.845	\\
0.375                     	&	0.1	& Gaussian         &	0.744 +/- 0.002	&	0.804/0.193	&	0.870/0.217	&	0.924/0.891	\\ \hline
0.0                       	&	1.0	& Gaussian         &	2.285 +/- 0.014	&	0.496/0.306	&	0.933/0.945	&	0.532/0.323	\\
0.0375                    	&	1.0	& Gaussian         &	1.991 +/- 0.011	&	0.494/0.268	&	0.929/0.694	&	0.532/0.387	\\
0.375                     	&	1.0	& Gaussian         &	0.595 +/- 0.002	&	0.463/0.160	&	0.870/0.366	&	0.532/0.437	\\ \hline
0.0                       	&	0.0	& Leptokurtic      &	5.689 +/- 0.010	&	0.933/0.929	&	0.933/0.929	&	1/1	\\
0.0375                    	&	0.0	& Leptokurtic      &	2.695 +/- 0.014	&	0.929/0.369	&	0.929/0.369	&	1/1	\\
0.375                     	&	0.0	& Leptokurtic      &	0.736 +/- 0.002	&	0.871/0.183	&	0.871/0.183	&	1/1	\\ \hline
0.0                       	&	0.1	& Leptokurtic      &	5.355 +/- 0.030	&	0.862/0.770	&	0.933/0.936	&	0.924/0.823	\\
0.0375                    	&	0.1	& Leptokurtic      &	2.637 +/- 0.019	&	0.859/0.360	&	0.929/0.420	&	0.924/0.858	\\
0.375                     	&	0.1	& Leptokurtic      &	0.721 +/- 0.002	&	0.803/0.181	&	0.870/0.206	&	0.923/0.881	\\ \hline
0.0                       	&	1.0	& Leptokurtic      &	3.377 +/- 0.045	&	0.498/0.283	&	0.933/0.938	&	0.534/0.302	\\
0.0375                    	&	1.0	& Leptokurtic      &	2.285 +/- 0.014	&	0.496/0.249	&	0.929/0.644	&	0.534/0.388	\\
0.375                     	&	1.0	& Leptokurtic      &	0.577 +/- 0.004	&	0.463/0.162	&	0.870/0.367	&	0.532/0.442	\\ \hline
\end{tabular}
\end{table}








\section{Discussion}


%broadly start discussion for where our study fits into the literature of this field
%why no range limits are found
%compare to polechova and barton 2015
%could bring in relation to bridle et al 2010

Range expansions are a unique demographic event that lead to an interesting suite of population genetic processes. These processes have been widely studied, yet examining the combination of both an environmental gradient over which expansion occurs and deleterious mutations that can contribute to expansion load has not previously been investigated. We found that under biologically realistic conditions, both expansion load and the process of adapting to an environmental gradient slow range expansion and lower the fitness of expanding populations. Previous theoretical studies focusing individually on either the effects of expansion load or the effects of load due to adaptation on a environmental gradient found similar results of reduced fitness at the range edge. Our finding that the incorporation of both of these scenarios simultaneously does not further reduce fitness is intriguing. That these factors interact to instead alleviate the reduction in fitness in expanding populations is both interesting and relevant to studies of expansion load and predicting the fate of many species subject to such expansions due to natural phenomena or perhaps more so in the future due to climate change.

Theory has shown that expanding populations can accumulate load due to deleterious mutations \citep{Peischl:2013, Peischl:2015, Peischl:2015b}, and this factor is even shown to cause range limits \citep{Peischl:2015b}. %What were the unrealistic assumptions of these models?
However, the extent of expansion load under realistic biological conditions merited further investigation, as elucidating the role of expansion load empirically is still highly studied today. Combining the presence of an environmental gradient with range expansions adds a new level of biological realism to studies of expansion load. Heterogeneous selection is experienced by species inhabiting multitudes of types of environmental gradients, be it latitudinal temperature change \color{red}(cite or change to another example) \color{black} or gradients in soil pH or nutrients \color{red}(cite)\color{black}. Populations expanding over environmental gradients are known to experience genetic load due to the influx of maladapted alleles from immigrants. This migration load has been shown to reduce fitness in expanding populations and play a role in creating stable range limits \citep{Bridle:2010, Polechova:2015}. %, and when genetic drift reduces genetic variance below a threshold level at which adaptation to spatially varying conditions fails \citep{}. 
Our study did not find the formation of stable range limits for the parameter sets investigated, but we report of the relative effects of all combinations of an environmental gradient and deleterious mutations in terms of their impact of the rate of range expansions and the reductions in fitness realized across the species range.


\subsection*{Presence and Amount of Expansion Load}

Our results corroborate previous studies, showing the presence of expansion load accumulating during a species range expansion. As expected, this load is higher with more mutational input. Perhaps unexpectedly, however, is that the scenario of most expansion load accumulated occurs in the absence of an environmental gradient, where a load of $39\%$ is observed. We can interpret this in the light of our results on expansion speed. The most striking effects on speed of expansion were due to the presence of an environmental gradient, where even a weak gradient substantially slowed range expansion. Because expansion load accumulates due to reduced population size and thus reduced efficacy of selection at the range edge, cases that expand the fastest will experience this effect to the greatest degree. There will be less time for recovery to occur from the core of the species range because fewer migrants will be able to successfully reach the expanding front. Therefore, seeing the greatest level of expansion load accumulate on our weakest gradient is intuitive. It is also interesting that the presence of even a weak environmental gradient is sufficient to substantially reduce expansion load.

The amounts of expansion load we observe are lower than those previously reported in theoretical studies. \citet{Peischl:2013} reported a load of approximately $75\%$ while\citet{Peischl:2015} found an approximate expansion load of $66\%$. These models differed in their assumption of hard and soft selection, where under soft selection, greater load can accumulate when competition is only local, while hard selection removes this effect when fitness is absolute. Our simulations also employ hard selection, but other parameter choices can also contribute to the differences found in terms of amounts of expansion load seen. Our choice of fecundity greatly impacts the speed at which populations expand across our landscape. Initial simulations with a fecundity of $3.5$ (data not shown) instead of our current mean fecundity of $7$, showed much less expansion load and higher fitness levels in edge populations. Increasing fecundity allows populations to persist at lower mean fitness and thus accumulate more load. The opposite is true under low fecundity situations where local population extinction occurs at higher fitness values and less load is able to accumulate. It is likely that if we were to increase fecundity further (which was not feasible computationally), even more substantial expansion load would be found.

% Empirical studies reported evidence of expansion load related to out-of-africa migration \citep{Lohmueller2008,Lohmueller2014,Do2015,Henn:2015}. \citet{Henn:2015} estimated maximal expansion load observed in humans. Depending on the model of dominance, these estimates vary between 1.7\% and 45\% of expansion load. Our simulations's results are more congruent to these data than previous simulations.

Measures of expansion load at a range edge may also differ in simulation studies depending on the degree of two-dimensionality modeled in landscapes. Much larger and wider landscapes may be more realistic for some biological systems, but are computationally difficult to model. We did not find any large, local deviations in fitness at the range edge in our two-dimensional simulations, but it is known that one- versus two-dimensional models yield slight differences in results for fitness reductions \color{red} think I recall someone saying this in a study, but need to find it \color{black}



\subsection*{Interaction of Environmental Gradient and Deleterious Mutations}

We find an intriguing interaction of the presence of an environmental gradient with the presence of deleterious mutations in our simulations. Rather than combining additively to further reduce fitness in edge populations, these factors instead interact to reduce load at the range edge and improve fitness. Steeper environmental gradients lead to higher fitness associated with deleterious mutations, and higher deleterious mutation rates lead to higher fitness associated with local adaptation to the environment. Though fitness was always lower at the range edge, there was less reduction when both an environmental gradient and deleterious mutations were present. This effect can be explained by several factors. 
% from point of view of gradient
When we consider the effects of heterogeneous selection from the presence of an environmental gradient, it is known from previous studies that the steeper a gradient, the more difficult expansion is. This is because local adaptation along the gradient is made more difficult by migrants creating an influx of maladaptive alleles \citep{Kirkpatrick:1997, Barton:2001, Polechova:2015}. This factor alone can reduce fitness at the range edge, in particular, due to the asymmetry of migration at a range edge where all migrants necessarily come from the core \citep{Kirkpatrick:1997}. The difficulty in adapting at the range edge, as previously mentioned, leads to a slower range expansion. Populations take more time to adapt across the gradient as it steepens. This time taken for local adaptation is key to the alleviation of expansion load.

Expansion load occurs as a result of gene surfing along the expanding front of low population sizes. As soon as these low population sizes have recovered to a larger $N_e$, selection again becomes effective at removing deleterious mutations. This process combines with the influx of migrants from even further towards the range core where selection has been able to act at eliminating deleterious variants for longer in time. When expansion is slowed, population sizes can recover more quickly, and migrants can reach more closely to the range edge, reducing the expansion load.

% from point of view of mutations
When we consider this effect from the point of view of expansion load, the effect is similar. Equilibrium mutation load in all of our scenarios with deleterious mutations already results in a reduced fitness across the species range (Figure \ref{fig:fitness}b). As we know, the expansion process increases load due to deleterious mutations at the range edge. This is a fitness reduction we do observe, and this reduction in fitness due to deleterious alleles has an effect on local adaptation of the quantitative trait. When fitness is reduced due to unconditionally deleterious mutations, for a given population to sustain itself and have an overall fitness greater than one, one (or both) of two things must happen. First, the population may persist as a low-fitness sink until selection and migration rescue the edge population from its accumulated deleterious alleles, or second, individuals may become better adapted to their local conditions to reach an overall fitness greater than one, despite the presence of deleterious alleles. We can see from Figure \ref{fig:allfreqs} and Supplemental files \color{red}Sn \color{black} that the first process of recovery and rescue is indeed occurring. This process takes time though, and for the improved local adaptation we see at the range edge across $U$ values and at any given gradient value, our results support the presence of this second process, where local adaptation is necessarily increased for populations to sustain.

% fecundity adjustments
Further evidence to support the presence of this second process of improved local adaptation comes from the simulations where mutation load was mimicked by fecundity reductions. When no deleterious mutations are present, but realized fecundity is forced to match that of equilibrium mutation load, fitness for the quantitative trait is still increased as ``mutation load" is increased. In other words, from Figure \ref{fig:fitness}a, at a given $b$ value, comparing $U = 0$ to the cases of no expansion load for $U = 0.1$ or $U = 1.0$ (dashed lines) improves fitness, and adding in the effects of expansion load on top of this improve quantitative trait fitness even further. Barring the case with no environmental gradient ($b = 0$), there are no differences in expansion speed between our simulations of ``no expansion load" and ``expansion load", further lending credit to the process of local adaptation causing these fitness improvements in the face of reduced fitness from deleterious mutations. 
%more quanti traits
Further investigations of this process may be fruitful, as it is clearly not biologically realistic for only one quantitative trait to determine local adaptation to the environment. Our results suggest that multiple quantitative traits adapting over similar or different environmental gradients may facilitate improved local adaptation to the environment.

%why kurtotic isn't a thing
Somewhat surprisingly, we found no effect of dispersal regime on the level of local maladaptation or expansion load in our simulations. Increased long distance dispersal with the kurtotic dispersal kernel did lead to faster range expansions, but this effect was only notable on a shallow or no environmental gradient. These differences in speed were not linked to differences in fitness, to which we attribute a link between for the Gaussian dispersal kernel. However, when considering that long distance dispersal only allows individuals to move farther, this does not imply that the change in speed this creates will correlate with a change in fitness. As discussed by \citet{Fayard:2009}, surfing was found to be less frequent under regimes of increased long distance dispersal because alleles are able to reach beyond the expanding front and establish, inhibiting accumulation of deleterious alleles that might be surfing and additionally improving genetic diversity at the edge. They also show that wider dispersal corridors can reduce the effect of surfing, which may or may not play a factor in our simulations, but presumably would have a similar effect across our modeled dispersal regimes. Furthermore, any increased speed by long distance dispersal that might be expected to increase expansion load due to faster expansion would be balanced out on an environmental gradient because dispersing further over a gradient is analogous to dispersing a shorter distance over a shallower gradient. With our results in mind, the increased maladaptation on an effectively steeper gradient would then act to slow expansion and alleviate expansion load to a level that could match the results for our Gaussian dispersal model.



\subsection*{Mechanism of Expansion Load}
% section on mechanism of expansion load (and detectability of expansion load in the real world)
We further investigated the effect sizes of deleterious mutations contributing to expansion load. The topic of effect sizes has large empirical implications, as many studies today aim to understand the presence of expansion load in humans after expansion out of Africa and the efficacy that selection has had on our species \citep{Henn:2015,Henn:2015b, Lohmueller:2014, Lohmueller:2014b, Gravel:2016}. The ability to detect expansion load versus the severity of its presence has been debated, and a full understanding of the selection coefficients across human (or other) genomes is little known \color{red}citations \color{black}.

Expansion load is attributed to larger effect alleles being able to accumulate under weakened natural selection, and this is a result our data support. Figure \ref{fig:loci} shows that most load is due to mutations of intermediate or larger effects. Small effect loci are easily able to fix not only at the range edge, so are prevalent, but have minimal impacts on fitness. We find an interesting difference in the make-up of expansion load between our two simulated cases of genome-wide deleterious mutation rates. For our lower mutation rate ($U = 0.1$, Figure \ref{fig:loci}a), the total expansion load is again greatest in the absence of an environmental gradient. Interestingly however, this increase in load is due more to mutations of relatively larger effects, where $s$ is greater than $\approx 0.2$. There are very few loci in general that exist at these effect sizes, but they are still able to fix at the edge (Figure \ref{fig:allfreqs} and cause a substantial portion of the expansion load. Furthermore, when compared to the case of $U = 1.0$ (Figure \ref{fig:loci}b), a qualitative difference can be seen in terms of which loci contribute most to expansion load. In this case, there is less inflation of relatively large mutations, and mutations of intermediate $s$ contribute the most. Interpreting these results in light of our expansion speed results leads us to conclude that as expansion speed increases (from steepest gradient to no gradient and from higher $U$ to lower $U$), mutations of relatively larger effects are able to accumulate at the range edge and contribute to expansion load.

%  \citep{Peischl:2015} studied the accumulation of expansion load due to recessive deleterious standing variation and found expansion load was driven mainly by deleterious alleles of mild and moderate effect (i.e., up to Ns < 2 for the parameter values used in their Figure 6).  %Are these effect sizes indeed smaller than what we found? If so, what are likely reasons for the difference?

\color{red}I think we'll save in depth comparison to the human results for another time \& paper \color{black}

%what are the implications for this in detecting expansion load in the real world?
%what do we think is a realistic detectable amount?
%what amounts have been shown in humans?
%are there any other systems that have looked at this? - I think there is some literature in bacteria or other microorganisms from experimental studies
%Can we compare to human papers which get different answers? (because dominance assumptions matter)

%Can we make useful insights for empiricists? ``Our results show that we can detect X amount of load due to loci of greater than or equal to effect size of Y." Based on our simulations, this means there is still Z\% of load that an empiricist might miss when using things like GERP etc (which tend to only work well at IDing larger effect deleterious muts).

%Lohmueller and Do each have separate papers in the human literature that have found different answers for load in europeans based on examining the data in different ways. compare this to what we can find from our data





\subsection*{Implications, Caveats, and Future Directions}%Biological Implications for Range Expansions}
%would be great to be able to tie this in not just to the human literature and the theory literature, but anything relevant to a wider audience. e.g. what are the implications for studies of local adaptation, and testing transplant of core vs edge pops, things like genetic rescue, etc

Our finding that local maladaptation interacts with expansion load has broad implications for studies of invasive species and conservation efforts. Though little study has been done on expansion load experienced in invasive species \color{red}(is there anything citeable?) \color{black}, invasive species are known generally to spread quickly upon successful establishment. Our results would indicate that such quick spread would lead to a greatly increased expansion load, perhaps suggesting that efforts to control invasives at their expanding fronts would only worsen their impact by stalling their expansion, reducing their load, and increasing their local adaptation. \color{red} (this is a very bold statement that I won't leave in unless you guys think it makes solid sense) \color{black} 

Conservation efforts that aim to reintroduce genetic diversity through assisted migration would clearly benefit edge populations in terms of reducing expansion load, but consideration would still be necessary in terms of potential to cause outbreeding depression \citep{Aitken:2013}. If climate change leads to increased occurrences of range expansion for many species, the impacts may be multifarious. If climate changes too quickly necessitating fast range expansion, expansion load will be increased and local adaptation decreased, a scenario that is unlikely to be biologically viable, or if so would leave populations in a very poor state and subject strongly to any stochastic extinction events. If the speed of climate change is not too fast however, adapting as populations move over space may reduce any potential impacts of expansion load.

There are several biological features of organisms and their environments that we did not consider in our simulations, and which merit future investigation as they may impact our results.  Our simulations allowed for individuals to self-fertilize under situations of mate-limitation, however this is not biologically possible in many species, especially animals. The inability to self-fertilize could slow range expansions, leading to a reduction in the expansion load. %Obligately outcrossing plants and dioecious species as well as species with lower fecundity would reduce propagule pressure at the leading edge \citep{Simberloff:2009, Hargreaves:2014}. 
Fecundity varies enormously across known species of the world, and as we have discussed, changing this parameter would result in quantitative differences in amountso f load found. Furthermore, Allee effects that reduce fitness in small edge populations \citep{Taylor:2005} or aggregating dispersal behavior that discourages colonization of empty habitat behavior may slow movement into empty or sparsely-occupied environments, and overall slow the speed of expansion. Dispersal barriers in the environment or increased encounters with antagonistic species (e.g. competitors, pathogens) could also slow or halt expansion \citep{Case:2005}.  Alternatively, other species traits could speed expansion beyond rates seen in our model, including further increased long distance dispersal. I would be interesting to expand the model to look at species with overlapping generations, where previously established individuals may block immigration into patches at carrying capacity (i.e. a priority effect). Priority effects would slow replacement of initial colonizers at the range edge, impeding genetic rescue and increasing the persistence of expansion load away from the edge. 

A potentially key evolutionary component of range expansions not included in our model is the evolution dispersal ability.  A survey of both simulation models and empirical evidence suggests that increased dispersal is always expected to evolve at expanding range margins, as dispersers both mate assortatively at the expanding edge and gain a fitness benefit by largely escaping intraspecific competition \citep{Hargreaves:2014}. Interestingly, increased dispersal is expected theoretically and found empirically even during expansion across environmental gradients to which populations are locally adapted. However, increased dispersal also steepens the perceived slope of a given environmental gradient, which can eventually slow or even temporarily halt range expansion until edge populations evolve to overcome initial maladaptation (e.g. \citealt{Phillips:2012}). It is thus foreseeable that reduced dispersal ability might evolve in our scenarios to suffer less of a fitness cost due to migrating to too different an environment. We would expect evolution of reduced dispersal to slow range expansions and alleviate expansion load.



\subsection{Conclusions}
% closing paragraph on broader implications


%Our results have implications for the fate of populations expanding or shifting their range in response to climate change. They suggest that the faster range expansion occurs, the more severe expansion load may be, due to the more frequent surfing of large effect deleterious alleles. Although we saw lower overall expansion load when range expansion occurred more quickly, this was due to a lower deleterious mutation rate. Climate change, on the other hand, may externally enforce a faster pace of range expansion than would otherwise result from mutational parameters, causing a larger buildup of expansion load. Future studies should examine the effects of expansion load and adaptation to a changing environmental gradient on shifts in and expansion and of species' ranges. Future studies should also examine how the combination of expansion load and adaptation to an environmental gradient affect the evolution of stable range limits. 

In conclusion, our results support those of previous studies finding that expansion load via the surfing of deleterious alleles reduces fitness in expanding populations \citep{Peischl:2013, Peischl:2015, Peischl:2015b}. We show this under biologically realistic conditions, bolstering evidence that allele surfing may, indeed, cause expansion load in nature. Our results are also in agreement with those of previous studies showing that on an environmental gradient, migration load reduces fitness in expanding populations \citep{Kirkpatrick:1997, Bridle:2010, Polechova:2015}. For the first time, we show that the presence of one of these types of load reduces the presence of the other by slowing range expansion and allowing time for evolutionary rescue. Finally, we demonstrate that faster range expansion leads to a larger contribution of moderate and large effect deleterious alleles to expansion load. These contributions significantly advance theory on the genetics of range expansion towards meaningful predictions and interpretations for studies of natural populations. 

%%% Local Variables:
%%% TeX-master: "thesis"
%%% TeX-PDF-mode: t
%%% End:
