\chapter{Introduction}
\label{chap:introduction}

% Yay inspiriational quotes.
\begin{quoteshrink}
% alternate quote: "The change from one stable equilibrium to the other may take place as the result of the isolation of a small unrepresentative group of the population, a temporary change in the environment which alters the relative viability of different types, or in several other ways..."
  ``I do not propose to argue the case for evolution, which I regard as being quite as well proven as most other historical facts, but to discuss its possible causes, which are certainly debatable.''
  \hfill\citet{Haldane:1932}, p.3
\end{quoteshrink}
% A Review of Some Fundamental Concepts and Problems of Population Genetics', Cold Spring Harbor Symposia on Quantitative Biology, 1955, 20, 13-14

\noindent


% great theme to come back to is the faults in simplified models of evolution

The study of evolution aims to understand the genesis and maintenance of the immense diversity in existence today. An essential part of this task requires understanding the genetic processes and patterns that occur within and among populations. Population genetics coalesced as a field in the early 1900s, following more than a decade of controversy and debate over the process of evolution \citep{Provine:2001}. Since its advent, great insight has been made into the processes that contribute to the evolution of populations and species. Our understandings of genetic drift, gene flow, mutation, and selection have created a solid foundation for the growing field of evolutionary biology. Yet as the field has advanced, realization that the manifold and minute complexities of population genetic processes can immensely impact the evolution of populations and species leaves many open questions in the field of evolutionary biology.

These questions can be addressed through theoretical studies, using analytic models or running simulations, or through empirical studies, gathering biological data for analysis. Both approaches to understanding evolution complement each other, and require each other. Population genetic theory uses simplified models that lead to evolutionary predictions and expectations that are then tested in the real world. Biology is always more complicated, and when results in the real world differ from our theoretical predictions, the process repeats and our understanding of biology grows. In my thesis, I take a simulation approach to further our understanding of population genetics in several interesting, non-equilibrium scenarios.

% demography
Demographic history of a population plays a major role in its evolution. Demography describes the characteristics of a population's size, distribution, and composition through statistics such as birth or death rates \citep{Pavlik:2000}. This information is informative for short-term predictions in populations, but in combination with population genetic data a more complete understanding of a population's past, present, and future can be attained, proving especially vital in conservation biology \citep{Lande:1988, Avise:1995, Nunney:1993}. Effective population size serves an exemplary description of this situation. Census size of many organisms can be obtained, as this simply requires counting all individuals present in a population (though for many reasons, census size can often be the more difficult quantity to obtain). However census size alone does not generally inform long term processes occurring in a population nor how the population may evolve. Genetic data is necessary for such insight. For example, a large number of individuals might be present at any given time due to stochastic events such as increase in resources, rather than reflecting consistently large numbers of individuals. Likewise a small number of individuals might be found if other stochastic abiotic factors such as a recent catastrophic storm lead to the death of many individuals. In these instances, defining a population by its effective size is more useful and can reveal information that the census size does not. 

% Ne
The effective population size is a key parameter in population genetics and evolution. Defined by \citet{Wright:1931} as the size of an ideal population that experiences the same level of genetic drift as the given population, effective population size informs how populations are likely to respond to evolutionary forces, or even what forces are likely to occur. The effective population size has been used widely as a deciding parameter in conservation situations such as suggesting minimally viable population sizes or establishing breeding populations in both plants and animals \citep{Lande:1987, Soule:1987, Ellstrand:1993}.

% Ne and demography -> gene flow
Estimating the effective population size for these purposes in natural population can, however, be a difficult task. To understand the degree of genetic drift a population experiences, and thus its effective size, drift must be measured from genetic data. This is generally accomplished by examining changes in allele frequencies over time. Genetic drift is only one process that has the potential to alter allele frequencies over time. Populations are subject to natural selection, mutations, and gene flow among populations -- processes that are all capable in altering allele frequencies in multitudes of ways. Methods developed for the purpose of estimating effective size attempt to work past some of these difficulties. Using only neutral loci removes the effects of selection, to the best of our ability for identifying which loci these are. Using data over very short times scales allows us to safely assume a lack of mutational input for the dataset at hand. The final process in this list, gene flow, is not as easily ignored.

Gene flow varies widely in amount across species, but is likely a prevalent force in many populations and species across the globe \citep{Slatkin:1985, Slatkin:1987}. 
Estimating gene flow and migration rates alone is already a daunting task possessing its own estimation methods and their accompanied difficulties \citep{Beerli:2001, Hey:2004, Kuhner:2006, Whitlock:1999}. Gene flow, can be a major disruptive force in evolution by introducing foreign and sometimes maladaptive individuals \citep{Slatkin:1987}. Whether migration into a population results in successful gene flow is linked to effective size in terms of whether selection is sufficiently strong to remove any maladaptive individuals introduced, or whether populations are so small that selection is weak and gene flow instead becomes a positive force acting to sustain an otherwise extinction-bound population by introducing genetic variation and increasing the effective size. Even within his controversial work on shifting balance theory, Sewall Wright recognized the importance of the interplay of effective population size and migration with the evolutionary process: %% maybe take this out b/c it's about shifting balance
\begin{quoteshrink}
  ``The rate of evolutionary change depends primarily on the balance between the effective size of population in the local strain and the amount of interchange of individuals with the species as a whole \dots''
  \hfill\citet{Wright:1930}
\end{quoteshrink}
This introduction of foreign alleles has the potential to alter allele frequencies on top of any levels of genetic drift a population is subject to, and disentangling the effects of these disparate processes can be difficult. The caveat introduced with the majority of estimators for effective population size is that the population must be isolated, but this assumption is likely violated in nature often. Knowing the demographic history of a population can thus help understand how the estimation process may be biased. Knowing the correct demographic history of a population can affect the ability of many genetic analysis methods to correctly infer a parameter of interest. For example, if a population under study recently underwent a vicariance event and was previously part of a much larger population, the genetic patterns seen may still reflect that of the larger population. Similarly, estimating immigration rates in such a population may lead to the incorrect inference of ongoing immigration when in fact this signal is only due to the recent similarity among the previously panmictic population. Making conservation decisions based on an incorrectly overestimated effective population size in this instance could be detrimental.

In \Chapref{effectivepopsize} I investigate the impact of migration on estimation of effective population sizes through genetic data analysis. Accurate estimation of the effective population size is key in conservation biology and leads to many political decisions in terms of when and where to conserve populations at risk and how many resources to dedicate to which causes. Misinterpreting or miscalculating effective size may thus lead to the local, or even global, extinction of species in the most extreme cases. Using simulated data and testing pre-existing methods developed for estimating effective population size allowed each of these methods to be equivalently tested across common demographic scenarios. Use of simulated data benefits this investigation by having knowledge of the true effective population size, which otherwise in nature is never truly possible.



% Ne and other things -> selection, drift, -> range expansions
% make this later as the link to range expansions:
Effective population size is relevant throughout many aspects of evolutionary biology for reasons other than conservation. Small populations are more subject to random genetic drift, while in larger populations selection is known to be more effective \citep{Gravel:2016}. Selection more successfully acts to remove deleterious genetic material and increase the frequency of beneficial alleles in populations when the effective size is larger. Effective population size thus impacts the likelihood of new mutations establishing or being lost within a population \citep{Kimura:1962, Whitlock:2000}. Larger effective populations sizes may even lead to greater ability of local adaptation \citep{Leimu:2008}.






%what even is a population? -> structure etc


% broad conclusion
Demographic history is a vital factor in the above processes. My thesis aims to investigate the interplay of demographic history with population genetic processes and patterns.

% start talking about gene flow






% not the best paragraph, but trying to link from Ne to local adaptation
Effective population size is a widespread parameter in many other facets of evolutionary biology. . And as described above, population size can determine the effective gene flow into populations, either inhibiting or promoting local adaptation. Local adaptation is a process that is highly studied in the context of the balance between gene flow and selection \citep{Kawecki:2004}.

A major goal of evolutionary biology today is to identify the loci that contribute to local adaptation of populations \citep{Savolainen:2013, Whitlock:2015, LeCorre:2012, Coop:2010}. Demographic history is, however, one of the leading causes of the misidentification of adaptive loci \citep{Whitlock:2015}(\color{red}other refs?\color{black}). Local adaptation is important in the process of evolution for many reasons. Adapting to the local environment is what allows many species to exist across a range of conditions. In some cases, local adaptation can also be considered an early stage in the process of speciation, contributing to differentiation among populations that eventually become reproductively isolated (\color{red}citations\color{black}). In today's world of climate change and other anthropogenic forces fragmenting or altering species' native environments, understanding the ability of populations to adapt is key to maintaining biodiversity. In addition to the utility of this knowledge in an applied sense, understanding the maintenance of genetic diversity across populations has been a longstanding question in evolutionary biology:
\begin{quoteshrink}
  ``The roles of local fitness peaks and gene flow in adaptive evolution remain major open questions in evolutionary biology, nearly a century after Wright first raised the issue.''
  \hfill\citet{Barton:2016}
\end{quoteshrink}


%% too much local adaptation stuff when my thesis is not about this
To best understand local adaptation, we rely on empirical data from natural species. Transplant studies and common gardens have contributed a great deal to identifying the presence of local adaptation within species. Finding the genetic loci that contribute to this adaptation is a complicated endeavor. Before considering the difficulties due to identifying the correct loci instead of the incorrect ones, there is an initial problem that is detecting loci contributing to a trait at all. Ample research has been devoted to finding whether adaptation is due mainly to few loci of large allelic effects or many loci of small effects (\color{red}citations\color{black}). Our current genetic and genomic technology combined with available statistical methods make the identification of small effect loci very difficult (\color{red}citations\color{black}). Therefore, finding few loci that contribute to an adaptive trait may be the correct biological answer, but may also be a detection error where small effect loci are missed as false negatives.

%% too much local adaptation stuff when my thesis is not about this - here too , just cut it down when linking to next section
Apart from detection issues, evolutionary processes themselves may lead to false inference of loci under selection. A common method for identifying loci important in local adaptation is by environment-allele correlations (\color{red}citations\color{black}). In these analyses, when certain alleles correlate to certain environmental conditions, they are predicted to be adaptive in those environments. Similarly, genetic clines over space that correlate with clines in environmental conditions are considered as adaptive. Inferring selection for these alleles is not unfounded, but recent research has shown that selection is not the only process capable of leading to these genetic patterns. Neutral processes such as genetic drift are equally possible causes for such a pattern, in particular when populations have non-equilibrium demographic histories. (\color{red}citations for this whole paragraph\color{black})


%% switch order of expansion chaps
A quintessential example of demographic history interacting with detection of loci under selection is that of species range expansions. Many species in the northern hemisphere have expanded northward after the last glaciation to recolonize previously uninhabitable terrain. This expansion process has left many populations in disequilibria. Range expansions can lead to a completely neutral genetic process that leaves a genetic signature in populations identical to that of selection on those loci. During range expansions, the expanding front continually colonizes new habitat. These front edge populations are thus consistently at lower effective population sizes. At the range edge, the strength of selection is reduced and genetic drift predominates. This can lead to the loss or fixation of alleles of various effects, beneficial or deleterious.  The repetition of this process during expansion has been termed gene surfing \citep{Klopfstein:2006}. As such alleles surf along the expanding front, this can leave behind the signature of a genetic cline, and this cline results from a purely neutral process during expansion. When such an expansion occurs over an environmental gradient, as is the case for northward expansion of species over latitudinal gradients in climate, the spatial autocorrelation of allelic clines with environmental clines can lead many inferences of loci under selection astray \citep{Whitlock:2015}.

Understanding the process of local adaptation during range expansions can contribute to our understanding of local adaptation in general. In \Chapref{heterogeneouslandscapes, expansionload} I present simulation studies examining this process. \Chapref{heterogeneouslandscapes} examines how range expansions over heterogeneous, patchy landscapes impact the ability of populations to adapt to the local environment. I test the interaction of this process with genetic architectures varying the presence of large effect mutations to further inform our understanding of adaptation via loci of small effect versus loci of large effect. Copious studies have investigated the process of range expansion and adaptation over environmental gradients \citep{Kirkpatrick:1997, Barton:2001, Bridle:2010, Polechova:2015, GarciaRamos:1997}. This theory has led to a strong understanding of the role of population sizes, migration, and gene flow at range edges. An important aspect of this process is asymmetric gene flow at a range edge. Because central populations tend to be more dense under the central-marginal hypothesis \citep{Brown:1984, Eckert:2008}, proportionally more gene flow reaches populations at a range edge. This gene flow can have one of two effects: swamping locally adaptive alleles with foreign, maladaptive alleles and preventing local adaptation, or introducing necessary genetic variation in otherwise depauperate populations thereby enhancing the ability to adapt. Investigations of these processes have used linear gradients in examining this process, while many environmental gradients in the real word may change more heterogeneously over space. Whether expansion over such heterogeneous landscapes better matches theory on adaptation in two patch models \citep{Gomulkiewicz:1995, Ronce:2001, Holt:1997, Gomulkiewicz:1999}, expansion over a linear gradient, or neither necessitates further investigation.

Furthermore, \Chapref{expansionload} examines a different aspect of range expansions -- the surfing of deleterious alleles at range edges leading to their accumulation and thus reductions in fitness. This accumulation of deleterious alleles during expansion is termed the expansion load. Theoretical studies have simulated this process and shown that these fitness reductions can be substantial in populations at the range edge \citep{Peischl:2013, Peischl:2015, Peischl:2015b}. This process has also been implicated in (and debated as) causing many of the deleterious variants seen in human populations due to their recent history of expansion our of Africa \citep{Henn:2015, Henn:2015b, Lohmueller:2008, Lohmueller:2014b, Do:2015}. We provide further insight into the process of expansion load by simulating the combination of deleterious alleles as well as adaptation across an environmental gradient to assess the impact and interaction of local adaptation to the environment with expansion load accumulated at the range edge.


In my final project, \Chapref{reproducibility} I present results from a project that has broad implications for all of my previous chapters. Understanding effective population sizes and populations undergoing range expansions necessitates that we know what a population itself is. This question is not as simple as it may sound, and determining what a true population is can be incredibly difficult in many systems \citep{Waples:2006}. Numerous methods exist and continue to be developed for the sole purpose of identifying populations or visualizing population structure in both continuous and discrete populations \citep{Pritchard:2000, Falush:2003, Falush:2007, Rosenberg:2004, Petkova:2015, Bradburd:2016}. To understand the performance of one such method, \textsc{structure}, we examined the reproducibility of results from \textsc{structure} analyses. This reproducibility study replicates published analyses, using the same datasets, to assess how biologically sound these inferences of population identification and delineation are. Not only was this a test of the ability to correctly infer a biological parameter, but made a substantial contribution to the field in terms of promoting data archiving and sufficient descriptions of methods by authors in order to make studies more repeatable and more reproducible. My thesis projects combined contribute to understanding the evolution of populations and species and how we can best measure and understand this evolution.

% add a paragraph to wrap up the intro at a broad level



%% - explain how knowing more about linkage and sweeps etc also matter with demography
%% - explain interaction of demog with each
%% - explain changing demog in relation to climate change etc- will be more common, so imp to study


%% A major question has emerged in the field of evolutionary biology and population genetics over the past decade: how do natural selection and past demographic history interact to shape current genetic diversity of populations and species? Researchers have come to realize that these two processes are not separate actions on the genome, but are instead intricately linked. Selective and neutral processes can occur concurrently and can often be incredibly difficult to disentangle. Recent studies1,2,3,4 provide examples of loci under selection that contribute to adaptation, and increasing amounts of genomic data allow for an improved understanding of the forces creating, maintaining, or eliminating genetic diversity. Disentangling the respective role of selection and demography is fundamental. Identifying loci that truly contribute to adaptation and determining when selection is most effective on the genome have implications for eliminating or combatting diseases, controlling invasive species, and ensuring the survival of species in the face of climate change and other anthropogenic environmental changes.

%% Demographic history contributes to every population's current genetic makeup. For example, population bottlenecks, demographic growth, admixture, or migration events all leave different genetic signatures in populations. Methods exist to infer such histories5,6, but it has recently been discovered that some demographic histories, particularly range expansions, can leave genetic signatures identical to others, confounding our ability to infer a population's past and what has contributed to its present and future. During a species range expansion, founding populations of small size at the expanding front experience a process termed gene surfing7, which can increase the frequency of neutral variants above expectation in newly colonized habitats, and look very much like an event of positive selection8. This same process can also result in the increase of deleterious variants that confer a mutational burden on populations and prevent their expansion or adaptation9 to new conditions due to the reduced efficiency of selection in small populations at range edges. The importance of these spatial expansions on the genetic diversity of populations has only recently been identified. Furthermore, a related demographic history of a range shift, where the entirety of a species range moves over space expanding at the front and receding at the trailing edge, has not been investigated in terms of its potential effects on population diversity. Furthering our understanding of these processes and their impact on species diversity will thus be a great advancement of the fields of evolutionary biology and population genetics.

%% 1) Stapley et al 2010 TREE 25:705-712, 2) Excoffier et al 2009 Heredity 103:285-298, 3) Stinchecombe and Hoekstra 2008 Heredity 100:158-170, 4) Foll and Gaggiotti 2008 Genetics 180:977-993, 5) Ramakrishnan et al 2005 MEC 14:2915-2922, 6) Hey and Nielsen 2004 Genetics 167:747-760, 7) Klopfstein et al 2006 MBE 23:482-490, 8) Edmonds et al 2004 PNAS 101:975-979, 9) Peischl et al 2013 MEC 22:5972-5982,  10) Tishkoff et al 2007 Nature Genetics 39:31-40




%%\section{Structure and contents of this thesis}



% version including adaptree?
% Population genetics as a field is experiencing a stimulating period of research. Genomic capabilities have advanced at astonishing speeds and continue to do so, providing easy access into insights previously beyond the reach of researchers studying non-model organisms. Simultaneously, the world is facing rapid environmental changes due to causes such as climate change and anthropogenic forces that result in habitat fragmentation, invasive species, or other harmful actions on species. Combined, these two factors mean that not only are we in desperate need of understanding the processes underlying evolution in changing environments to preserve biodiversity into the future, but we are also increasingly able to accomplish exactly these goals. However, despite the increasing ease of data generation, inferential methods and analyses require analogous advances to succeed in expanding our knowledge on evolutionary processes and patterns.
% In this thesis, I present my research which has touched in part on all of these topics. I have assessed the capabilities of methods for estimating effective population size to function accurately in the presence of migration, a factor which can skew estimation results in unexpected ways due to allele frequency changes. I have performed a validation study of genetic loci identified through genome wise associations and environment-allele correlations to test for the presence of false positives. And I have performed simulation studies to elucidate the intricacies of species range expansions and the role of heterogeneously changing environmental gradients, mutation load, and migration load on preventing or enhancing range expansions.

%Population size is a defining characteristic in countless ways. Census size of populations at the broadest level can describe the density at which species exist. Yet often, even describing what a population itself is can be a difficult task. One of the most essential parameters in evolutionary biology however, is the effective population size. This measure of a population is defined by the amount of genetic drift a population experiences. It describes the genetic diversity present, and thus how likely the population is to persist into the future. Effective population size further relates to many major evolutionary processes







%%% Local Variables:
%%% TeX-master: "thesis"
%%% TeX-PDF-mode: t
%%% End:
